.nuget/NuGet.Config

```diff
@@ -5,6 +5,6 @@
   </solution>
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="Storm Commerce" value="http://storm-build.prod.enferno.se/nuget" />
+    <add key="Storm Commerce" value="https://storm-build.prod.enferno.se/nuget" />
```

Comment:
Här ska vi använda den nya NuGet servern på Azure. Storm Commerce använder vi inte längre.
        <add key="StormPackageFeed" value="https://pkgs.dev.azure.com/stormcommerce/_packaging/StormPackageFeed/nuget/v3/index.json" />

Reply:
should be fixed now